lib/bap_disasm/bap_disasm_x86_lifter.ml

```diff
@@ -0,0 +1,2319 @@
+(** Native lifter of x86 instructions to the BAP IL *)
+
+open Core_kernel.Std
+open Bap_types.Std
+(* open Bil *)
+module BZ = Big_int_Z
+(* open Type *)
+(* open Arch *)
+(* open Var *)
+
+module BV = Bitvector
+
+module Util = struct
+  let id x = x (* useful for tabulating lists, for instance. *)
```

Comment:
There is `ident` function, and `Fn.id` functions, available from `core`, as soon as you opened `Core_kernel.Std`. So you don't need this